hello.html

```diff
@@ -0,0 +1 @@
+<h1>Hello Microverse!</h1>
```

Comment:
- [ ] I think you should create HTML code You can follow this [page](https://www.w3schools.com/html/html_editors.asp) so how to create HTML page and then you can write your heading inside it and don't forget to attach the stylesheet file so the chnages happens on the h1 tag element 🙏🏻🔥😊